python-package/lightgbm/engine.py

```diff
@@ -383,7 +387,8 @@ def cv(params, train_set, num_boost_round=100,
     Parameters
     ----------
     params : dict
-        Parameters for Booster.
+        Parameters for Booster. If callable objective functions is passed through ``params`` and ``fobj``
+        the Booster uses the first one.
```

Comment:
```suggestion
        Parameters for Booster. Values passed through `params` take precedence over those
        supplied via keyword arguments.
```

It's ok with me to add documentation on the `params` argument here, but I think it should be more general (since that behavior isn't specific just to `objective`), and should be more explicit about params vs. keyword arguments (as "the first one" is unclear).

Reply:
got it